File: java/src/test/java/util/AdventUtilsTest.java
package util;

import org.junit.jupiter.api.Test;

import java.util.List;

import static org.junit.jupiter.api.Assertions.assertNotEquals;

class AdventUtilsTest {

    @Test
    void shouldParseLinesOfTextFile() {
        // When
        final List<String> lines = AdventUtils.parseInput(2023, 1);

        // Then
        assertNotEquals(0, lines.size());
    }
}

File: scala/src/test/scala/util/AdventUtilsTest.scala
package util

import org.scalatest.funsuite.AnyFunSuite

class AdventUtilsTest extends AnyFunSuite {

  test("should parse lines of a text file") {
    // When
    val lines: List[String] = AdventUtils.parseInput(2024, 1)

    // Then
    assert(lines.nonEmpty)
  }
}
